app/models/google_event.rb

```diff
@@ -111,6 +105,20 @@ def load_emails
       ConferenceRoom.pluck(:email)
     end
 
+    GOOGLE_EVENT_DECLINED_RESPONSE = 'declined'.freeze
+    def add_events_from_room(room, service, events, config)
+      service.list_events(room.email, config) do |result, _|
+        if result
+          result.items.each do |event|
+            next if event.attendees.find(&:self).response_status == GOOGLE_EVENT_DECLINED_RESPONSE
+            normalize_dates(event)
+            events[event.start.date_time.wday] ||= []
+            events[event.start.date_time.wday] << event.to_h.merge(conference_room: room)
```

Comment:
Serialization. Almost whole logic with returned object is done by JavaScript anyway.